src/ui/components/SecondaryToolbox/index.js

```diff
@@ -88,17 +88,19 @@ function InspectorPanel() {
   );
 }
 
-function SecondaryToolbox({ selectedPanel, setSelectedPanel, narrowMode }) {
+function SecondaryToolbox({ selectedPanel, setSelectedPanel, narrowMode, recordingTarget }) {
   const {
     userSettings: { show_react },
   } = hooks.useGetUserSettings();
+  const isNode = recordingTarget == "node";
```

Comment:
should this be using triple equals?
```suggestion
  const isNode = recordingTarget === "node";
```

Reply:
hehe